RZSequenceController.h

```diff
@@ -0,0 +1,208 @@
+//
+//  RZSequenceController.h
+//
+//  Created by Rob Visentin on 1/12/15.
+
+// Copyright 2014 Raizlabs and other contributors
+// http://raizlabs.com/
+//
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+//
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+@import UIKit;
+#import "RZSequenceChildViewController.h"
+
+/**
+ *  The direction of an RZSequenceController.
+ */
+typedef NS_ENUM(NSUInteger, RZSequenceDirection){
+    /**
+     *  The sequence pans from side to side.
+     */
+    RZSequenceDirectionHorizontal = 1,
+    /**
+     *  The sequence pans up and down.
+     */
+    RZSequenceDirectionVertical
+};
+
+@class RZSequenceController;
+
+/**
+ *  An RZSequenceControllerDataSource is responsible for providing an RZSequenceController with each view controller in the sequence,
+ *  as well as with transitions between view controllers in the sequence.
```

Comment:
Is it worth adding that the data source is responsible for instantiating the VCs it provides?